multi-arch-builders/coreos-s390x-rhcos-builder.bu

```diff
@@ -24,7 +24,7 @@ systemd:
         Description=Create secex-data volume
         [Service]
         Type=oneshot
-        ExecStart=/home/core/create-secex-data.sh
+        ExecStart=/usr/local/bin/create-secex-data.sh
```

Comment:
maybe we should add `RemainAfterExit=yes` here since this is a oneshot unit.

Reply:
Well it does change the state of the system, so it should be the best practice. I will update the PR.